sdk/core/Azure.Core/src/Pipeline/RetryOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Pipeline.Policies;
+
+namespace Azure.Core.Pipeline
+{
+    public class RetryOptions
+    {
+        /// <summary>
+        /// Gets or sets the maximum number of retry attempts before giving up.
+        /// </summary>
+        public int MaxRetries { get; set; } = 3;
+
+        /// <summary>
+        /// Gets or sets the timespan used as delay between the retries or as a base for exponential backoff.
+        /// </summary>
+        public TimeSpan Delay { get; set; } = TimeSpan.FromSeconds(0.8);
+
+        /// <summary>
+        /// Gets or sets maximum timespan to pause between requests.
+        /// </summary>
+        public TimeSpan MaxDelay { get; set; } = TimeSpan.FromMinutes(1);
+
+        /// <summary>
+        /// Gets os sets retry mode
+        /// </summary>
+        public RetryMode Mode { get; set; } = RetryMode.Fixed;
```

Comment:
Is there a concept for how this would map to a custom retry policy that a developer may want to apply?   Or is the plan that the options wouldn't play into that?

Reply:
> Or is the plan that the options wouldn't play into that

Correct. Options are for default policies. Custom policy implementor might choose to interpret options but is not required to.